storage/StorageImpl.cc

```diff
@@ -62,18 +62,24 @@ namespace storage
 
 	Devicegraph* probed = create_devicegraph("probed");
 
+    // FIXME
+    // This approach for vgs activation does not work in combination with activating
+    // LUKS. Managing lvm with LUKS, raid, etc is pending to implement.
 	switch (environment.get_probe_mode())
 	{
 	    case ProbeMode::STANDARD: {
+	    LvmVg::Impl::activate();
```

Comment:
Wrong indentation - please fix

Please notice that a tab is 8 blanks (!) according to our coding style. Everything else just breaks everybody's editor.